filebeat/crawler/prospector.go

```diff
@@ -97,35 +100,89 @@ func (p *Prospector) Run(wg *sync.WaitGroup) {
 				return
 			case event := <-p.harvesterChan:
```

Comment:
what kind of filtering? I think generic filtering will be hidden within libbeat.